### DISCUSSION TITLE
Welcome to liquid-dsp Discussions!

### DISCUSSION THREAD

**Discussion by jgaeddert** [jgaeddert/liquid-dsp#262] [Announcements]
## 💦 Welcome!

I'm trying out Discussions as a place to connect with other members of this community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fsheikh** [comment]
@jgaeddert thanks for this initiative. As a way of introduction, I am a system programmer currently working out of southern Germany. My interest in Liquid stem from a [thesis](http://prr.hec.gov.pk/jspui/bitstream/123456789/2059/1/704S.pdf) I wrote during college days about multi-rate filters application in software/cognitive radios. Looking forward to positive interactions with liquid's community.

**Reply by jgaeddert** [reply]
@fsheikh thanks for the introduction! I hope these discussions take off. Your thesis is certainly right up my alley, of course!
